### PR TITLE
Updating readme to reflect bot setup and execution

### PR DESCRIPTION
**Pull request by saf6260**
Adding more infos about how this all works

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,41 @@
...
+
+## .Env File
+This file is used to hold all of the important information in regards to the
+bot id and channel ids for execution. Currently there are 4 values that need
+to be defined:
+* BOT_TOKEN -> The identification token of the bot. Discuss with the current
+  tech head for information on what the test bot's id is. NEVER share the bot
+  token to anyone you don't trust
+* BOT_ID -> The identification number of the bot. If setup properly,
+  this value can be obtained by right clicking on the channel and selecting
```
Comment (mjschuetze102) [inline README.md]: Switch channel to bot

File: README.md [hunk 2]
```diff
@@ -1,2 +1,41 @@
...
+  `COPY ID`
+* ANNOUNCEMENT_ID -> The id of the announcement channel. This can also be
+  obtained in the same way as the BOT_ID
+* VOICE_PARENT_ID -> The id of the voice/text category for mentees (where all
```
Comment (mjschuetze102) [inline README.md]: Maybe clarify this is a channel heading

Comment (mjschuetze102) [reply README.md]: Category is a bit ambiguous

**Review by mjschuetze102 — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (saf6260) [reply README.md]: Good call, added further clarification here

File: README.md (hunk 1 above)
Comment (saf6260) [reply README.md]: Good catch, adjusted here to say bot, and then channel or channel header for the following lines

**Review by mjschuetze102 — APPROVED** [review]

**Pull request merged**